index.js

```diff
@@ -35,29 +38,42 @@ const getWslDrivesMountPoint = (() => {
 
 		let isConfigFileExists = false;
 		try {
-			await pAccess(configFilePath, fs.constants.F_OK);
+			await fs.access(configFilePath, fs.constants.F_OK);
 			isConfigFileExists = true;
-		} catch (_) {}
+		} catch {}
 
 		if (!isConfigFileExists) {
 			return defaultMountPoint;
 		}
 
-		const configContent = await pReadFile(configFilePath, {encoding: 'utf8'});
-		const configMountPoint = /root\s*=\s*(.*)/g.exec(configContent);
+		const configContent = await fs.readFile(configFilePath, {encoding: 'utf8'});
+		const configMountPoint = /root\s*=\s*(?<mountPoint>.*)/g.exec(configContent)?.groups?.mountPoint?.trim();
 
 		if (!configMountPoint) {
 			return defaultMountPoint;
 		}
 
-		mountPoint = configMountPoint[1].trim();
-		mountPoint = mountPoint.endsWith('/') ? mountPoint : mountPoint + '/';
+		mountPoint = configMountPoint.endsWith('/') ? configMountPoint : `${configMountPoint}/`;
 
 		return mountPoint;
 	};
 })();
 
-module.exports = async (target, options) => {
+const pTryEach = async (array, mapper) => {
+	const errors = [];
+
+	for await (const item of array) {
+		try {
+			return await mapper(item);
```

Comment:
The `return` should be kept because we are looking for the value of the first call that succeeds.